docs/glossary.tsv

```diff
@@ -0,0 +1,46 @@
+TERM	DESCRIPTION	Example	editor	source	synonym
+Common Data Element	A common data element are standardized key terms or concepts, established so that they may be used in studies, to enhance data quality and reusability over time.	Age,Gender,LogP,IC50	Yojana	WikiPedia (https://clinfowiki.org/wiki/index.php/Common_Data_Element_(CDE)#:~:text=Common%20Data%20Element%20(CDE)%20are,across%20sites%20and%20over%20time.	CDE
```

Comment:
```suggestion
Common Data Element	A common data element is a standardized key term or concept, established so that it may be reused in a range of projects/studies, to enhance data quality and reusability over time.	Age,Gender,LogP,IC50	Yojana	WikiPedia (https://clinfowiki.org/wiki/index.php/Common_Data_Element_(CDE)#:~:text=Common%20Data%20Element%20(CDE)%20are,across%20sites%20and%20over%20time.	CDE
```